tmk_core/rules.mk

```
@@ -431,6 +436,14 @@ check-size:
 	$(SILENT) || echo "(Firmware size check does not yet support $(MCU) microprocessors; skipping.)"
 endif
 
+MD5SUM_CMD = md5sum
+ifneq ($(filter Darwin FreeBSD,$(shell uname -s)),)
+  MD5SUM_CMD = md5
+endif
```

```suggestion
MD5SUM ?= md5sum
ifneq ($(filter Darwin FreeBSD,$(shell uname -s)),)
  MD5SUM = md5
endif
```
And should probably be moved up around L205.

---

Done.